tutorials/katacoda/thanos/1-globalview/step2.md

```diff
@@ -53,7 +53,7 @@ docker run -d --net=host --rm \
     -v $(pwd)/prometheus0_eu1.yml:/etc/prometheus/prometheus.yml \
     --name prometheus-0-sidecar-eu1 \
     -u root \
-    quay.io/thanos/thanos:v0.19.0 \
+    quay.io/thanos/thanos:v0.20.0 \
```

Comment:
Is this planned? (full version not RC?)-  I think it's fine as we merge it to main and only them it will be used by Katacoda

Reply:
Let me use rc to be on the safer side.

Reply:
what happens with users using `latest` ?